shared/packages.sh

```diff
@@ -62,17 +62,17 @@ WILDMIDI_URL="https://github.com/Mindwerks/wildmidi/archive/wildmidi-0.4.5.tar.g
 WILDMIDI_DIR="wildmidi-wildmidi-0.4.5"
 WILDMIDI_ARGS="-DWANT_PLAYER=OFF -DWANT_STATIC=ON"
 
-OPUS_URL="https://downloads.xiph.org/releases/opus/opus-1.4.tar.gz"
+OPUS_URL="https://downloads.xiph.org/releases/opus/opus-1.5.1.tar.gz"
 OPUS_ARGS="--disable-intrinsics --disable-extra-programs"
-OPUS_DIR="opus-1.4"
+OPUS_DIR="opus-1.5.1"
```

Comment:
This one above...

Reply:
oh we had already a global flag for the intrinsics. Yes it is different.

Intrinsics are compiler-specific SIMD functions like (taken from the configure.ac directly) ``_mm_set1_ps((float)time(NULL));``. ASM is inline assembly.

Though no idea why they even splitted this into two options. It doesn't really matter if you write inline-assembly or call intrinsics. Both have the goal of maximizing the speed via platform-specific machine code 🤷